README.md

```diff
@@ -55,3 +55,7 @@ User Interactions
 ## Developer docs
 - [Frontend](./frontend/README.md)  
 - [Backend](./backend/README.md)
+
+Setup
+1. Clone this repository.
+2. Run `git submodule update`
```

Comment:
@Sigfried I can't remember the exact setup and update steps, but I put this here for now.

I have another computer I often like to use. Not sure why my public key is failing. I thought it was the same. I'll double check.

```
git submodule update
Cloning into '/Users/joeflack4/projects/ValueSet-Tools/termhub-csets'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:jhu-bids/termhub-csets.git' into submodule path '/Users/joeflack4/projects/ValueSet-Tools/termhub-csets' failed
Failed to clone 'termhub-csets'. Retry scheduled
Cloning into '/Users/joeflack4/projects/ValueSet-Tools/termhub-vocab'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:jhu-bids/termhub-vocab.git' into submodule path '/Users/joeflack4/projects/ValueSet-Tools/termhub-vocab' failed
Failed to clone 'termhub-vocab'. Retry scheduled
Cloning into '/Users/joeflack4/projects/ValueSet-Tools/termhub-csets'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:jhu-bids/termhub-csets.git' into submodule path '/Users/joeflack4/projects/ValueSet-Tools/termhub-csets' failed
Failed to clone 'termhub-csets' a second time, aborting
```